### DISCUSSION TITLE
Possible Values for CurrentStateString?

### DISCUSSION THREAD

**Discussion by SaltySOMAdmin** [rodneyviana/ODSyncUtil#22] [Q&A] [answered]
Thanks for creating this! Our org has required MFA from certain subnets and it's broken the connection to OneDrive for some devices. Is there anyway to find all of the available CurrentStateStrings? I don't have a device ready in this state (Where OneDrive was syncing but is now waiting for a MFA Prompt) to test with yet, though I'm sure they exist and the user is unaware.

**Comment by rodneyviana** [comment, marked as answer]
|Code|Status|
|---|---|
|0|Synced|
|1|Syncing|
|3|Paused|
|4|Error|
|5|Offline|

**Reply by Runitupdad** [reply]
I have an issue that i encounter, where users end up with their one drive having a red X over the blue one drive icon. I can't figure out how to make it happen, but it once they click the icon it asks to sign in with the shiboleth and non-msft two factor. (it works as normal after that)

Do you know if this will show up as 4 or 5 or if it'll just return nothing? (If you don't know I'll try to remember to report back when I encounter one again)

**Reply by SaltySOMAdmin** [reply]
I don't recall, I believe I had my script running this exit with the code if code != 0 or 1 though.
